lld/test/ELF/riscv-relax-gp-partial-align-min.s

```diff
@@ -0,0 +1,40 @@
+# REQUIRES: riscv
```

Comment:
Do we need `lld/test/ELF/riscv-relax-gp-*` files? Can they be folded into two files or even one?

Reply:
I was actually wondering the same. I don't have much experience writing these. I'll give it a shot.